### ISSUE TITLE
Cannot open configuration file after restart of docker image

### ISSUE THREAD

**Issue by OpieO1** [DyonR/docker-qbittorrentvpn#56]
I am able to successfully start and connect with the docker image on the initial deployment. However after restarting the image, it provides an error of:

> Options error: In [CMD-LINE]:1: Error opening configuration file: /config/openvpn/us_east-aes-128-cbc-udp-dns.ovpn

I have identified that my user is uid 1026 and my group id is 100. As a result I have used the following for my docker-compose:
```

version: '2.2'
services:
 qbittorrent:
  image: dyonr/qbittorrentvpn
  ports:
    - "8999:8999"
    - "8999:8999/udp"
    - "8090:8080"
    - "8118:8118"
 
  volumes:
    - /volume1/docker/qbittorrentvpn/config:/config
    - /volume1/docker/qbittorrentvpn/downloads:/downloads
  environment:
    - PUID=1026
    - PGID=100
    - VPN_ENABLED=yes
    - VPN_TYPE=openvpn
    - LAN_NETWORK=192.168.2.0/24
  restart: unless-stopped
```
The first time the container is deployed, the container will connect properly with the configuration file in place in the config/openvpn directory. However after a restart it always fails on opening the configuration file. It appears to be a permissions issue, but I'm not quite sure how to address the issue. Are there specific permissions I need to set on the ovpn file?

**Comment by DyonR** [comment]
This issue seems related to https://github.com/DyonR/docker-Jackettvpn/issues/50 and the main issue https://github.com/haugene/docker-transmission-openvpn/issues/1839  
  
Check out this comment on the docker-transmission-vpn section: https://github.com/haugene/docker-transmission-openvpn/issues/1839#issuecomment-894724920  
  
Please let me know if that also fixed it for you

**Comment by SheepKid12** [comment]
I'm seeing the same issue as @OpieO1 (except for me, it doesn't connect on first launch of the container) when attempting to use configs from ProtonVPN (running on a Synology DS1520+ running DSM 7). Looking at Opie's config, it does look like they're also on a Synology (due to the `volume1` volumes).

I did try the fix from the transmission issue you mentioned, and it didn't fix the issue.

Here's what I specifically added to the docker-compose:
- The `NET_ADMIN` & `network_mode` parts from the transmission issue referenced
- `privileged: true` (since I was seeing some errors with `/dev/net/tun` not working correctly).

I've also tried changing the PUID & PGID to match my user's, and no luck. I also tried to give permissions to all users using DSM's file manager. Still no luck with that.
